# 
In this respository I gathered some of my work from my time as a scientific assistant (Hiwi) with Dr. Johannes Wirth. It's supposed to serve as a first orientation on how to work on the data with certain methods. I have mainly worked with the following Python packages:

1. Scanpy: https://scanpy.readthedocs.io/en/latest/tutorials/index.html
   
   Mainly used to preprocess, cluster and visualize data.
3. Graphcompass: https://github.com/theislab/graphcompass/tree/main/notebooks

   Study spatial organization across conditions at different levels of abstraction.
5. Monkeybread: https://monkeybread.readthedocs.io/en/latest/

   Focus was on niche analysis tools. Tested on Glioblastoma data.
6. Comparison between two lymphnode datasets from 10xGenomics: In situ gene expression datasets with xenium onboard analysis 3.0.0 (testing of 5000 genes) and 1.5.0 (testing of 400 genes).

   - 3.0.0: https://www.10xgenomics.com/datasets/preview-data-xenium-prime-gene-expression
   - 1.5.0: https://www.10xgenomics.com/datasets/human-lymph-node-preview-data-xenium-human-multi-tissue-and-cancer-panel-1-standard

